src/coupled/ESM1.5/casa_um_inout.F90

```diff
@@ -507,8 +507,6 @@ SUBROUTINE casa_reinit_pk(casabiome,casamet,casapool,casabal,veg,phen, &
   wresp_n = 0.
   wresp_p = 0.
 
-  thinning = 0.
```

Comment:
`thinning` is the thinning fraction forcing data. This is the variable that is passed all the way through the UM code to here and is loaded from the restart file. The run script must modify the restart file, much like the tile fractions for this to change. We don't want it initialized to 0 because that would overwrite the data loaded from the restart file.

Reply:
sorry misread the diff. assumed the "-" was a "+"